learn/advanced/updating.md

```diff
@@ -225,35 +176,192 @@ Once the `dumpCreation` task shows `"status": "succeeded"`, you're ready to move
 
 ## Step 4: Stop the Meilisearch instance
 
-Stop your Meilisearch instance. If you're running Meilisearch as a `systemctl` service, you can stop your instance with:
+Stop your Meilisearch instance. If you're running Meilisearch as a `systemctl` service, connect via SSH to your cloud instance and execute the following command to stop Meilisearch:
 
 ```bash
 systemctl stop meilisearch
 ```
 
-## Step 5: Delete the database folder
+You may need to prefix the above command with `sudo` if you are not connected as root.
+
+## Step 5: Create a backup
 
-To delete the old Meilisearch version, you need to delete the `data.ms` folder. `data.ms` should be at the root of the Meilisearch binary, unless you chose [another location](/learn/configuration/instance_options.md#database-path).
+Instead of deleting `data.ms`, we suggest creating a backup in case something goes wrong. `data.ms` should be at the root of the Meilisearch binary, unless you chose [another location](/learn/configuration/instance_options.md#database-path).
 
 ::: note
 If you are using the Meilisearch official images on DigitalOcean, AWS, or GCP, you will find the `data.ms` folder at `/var/lib/meilisearch/data.ms`.
 :::
 
-## Step 6: Import the dump
+Move the binary of the current Meilisearch version and the database to the `tmp/` folder:
+
+:::: tabs
+
+::: tab Local installation
+
+```
+mv /path/to/your/meilisearch/directory/meilisearch /tmp
+mv /path/to/your/meilisearch/directory/meilisearch/data.ms /tmp/
+```
+
+:::
+
+::: tab Cloud platforms
+
+```
+mv /usr/bin/meilisearch /tmp
+mv /var/lib/meilisearch/data.ms /tmp/
+```
+
+:::
+
+::::
+
+## Step 6: Install the desired version of Meilisearch
+
+Install the latest version of Meilisearch using:
+
+:::: tabs
+
+::: tab Local installation
+
+```bash
+# Install Meilisearch
+curl -L https://install.meilisearch.com | sh
+
+# Launch Meilisearch
+./meilisearch
+```
+
+:::
+
+::: tab Cloud platforms
+
+```sh
+# replace {meilisearch_version} with the version of your choice. Use the format: `vX.X.X`
```

Comment:
Should also replace `meilisearch-linux-amd64` with the appropriate OS, no?

Reply:
Good catch!

Reply:
[Most cloud platforms run on Linux](https://www.redhat.com/en/resources/state-of-linux-in-public-cloud-for-enterprises) but I will confirm